### DISCUSSION TITLE
Progenerated Case Files

### DISCUSSION THREAD

**Discussion by xqwerty151x** [joshreve/dactyl-keyboard#29] [Q&A]
Howdy, I was wondering if there were any progenerated case files anywhere. I have been trying to run the python scripts and have not been able to do so. I keep getting a ModuleNotFoundError: No module named 'cadquery'  error, or if I replace it with the "solid engine it is the same except with solid. 
Maybe I am doing something wrong but I have reinstalled everything twice and asked a friend who has some experience with python for help but he could not figure it out. 
If anyone can help it would be greatly appreciated.

**Comment by joshreve** [comment]
There are pre-generated files in the releases section.  There got to be too many that were too large to keep with the code.  Check the releases and there are a long list of configs to look at.  There aren't files for the trackball yet as it is not stable.  I will add them to the releases when I complete the work and merge with master.

Have you tried the Docker creator?  Also, have you done the install for cadquery or solid python?  If you check the readme it has the easiest way to install.
